src/app/components/form/input/FormInputPassword.vue

```diff
@@ -11,9 +11,20 @@
     <template v-if="isStrengthShown" #inputSuffix>
       <Progress :model-value="strength" class="my-2" />
     </template>
+    <template #clearButton>
+      <ButtonIcon
+        :aria-label="t('iconAltClose')"
+        class="hidden flex-shrink-0 px-2 peer-[:not([data-empty='true'])]:block peer-[&[aria-invalid=true]]:text-(--semantic-critic-text)"
+        @click="emit('input', '')"
+      >
+        <IVibetypeClose :aria-label="t('iconAltClose')" class="size-6" />
+      </ButtonIcon>
+    </template>
     <template #icon>
-      <IHeroiconsEye v-if="!isVisible" />
-      <IHeroiconsEyeSlash v-else />
+      <div class="flex flex-row gap-1">
+        <IHeroiconsEye v-if="!isVisible" />
+        <IHeroiconsEyeSlash v-else />
+      </div>
```

Comment:
Is the wrapping `<div class="flex flex-row gap-1">` needed or a leftover that can be removed?